packages/plugin/src/Bundles/Backup/Controllers/ImportController.php

```diff
@@ -25,6 +27,19 @@ public function actionExpressForms(): Response
         return $this->asSerializedJson($data, context: ['preserve_empty_objects' => false]);
     }
 
+    public function actionFormie3(): Response
+    {
+        try {
+            $propertyProvider = \Craft::$container->get(PropertyProvider::class);
+            $exporter = new FormieV3Exporter($propertyProvider);
+            $data = $exporter->collectDataPreview();
```

Comment:
When using the Dependency Injection Container you don't need to get the property provider, the injector will take care of it.

```suggestion
        try {
            $exporter = \Craft::$container->get(FormieV3Exporter::class);
            $data = $exporter->collectDataPreview();
```